perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -196,13 +202,25 @@ export default {
       return this.cantBuyProduct ? '' : 'clickable ';
     },
     cantBuyProduct() {
-      return (this.disabledBuy || !this.walletEnabled || this.walletLoading) || !this.displayBuyButton;
+      return (this.disabledBuy || !this.walletEnabled || this.walletLoading) || !this.displayBuyButton || (this.product.price > this.balance);
```

Comment:
you are sure of this behavior ?
you have to check it with @mzorai

Reply:
checked and it's a new behavior!